packages/cli/test/cli-codemod.spec.ts

```diff
@@ -16,13 +17,31 @@ describe('Stylable Cli Code Mods', () => {
     it('apply all code mods when no specific filter applied', () => {
```

Comment:
this doesn't seem to no longer describe what the test does

Reply:
true